### PR TITLE
Rembert onnx support

### LINKED ISSUE #2092: Add support for RemBERT in the ONNX export
### Feature request

Add RemBERT to supported architectures for ONNX export.

### Motivation

The support for [RemBert](https://huggingface.co/docs/transformers/model_doc/rembert) was previously available in Transformers see [here](https://github.com/huggingface/transformers/issues/16308). However,  now it seems that RemBERT is no longer supported.

### Your contribution

I can help by testing implementation or providing the code if provided by some tutorial. I was not able to find documentation on how to do that.

### PR DESCRIPTION
**Pull request by mlynatom**
# What does this PR do?

Fixes #2092

Hi, this pull request adds support for the RemBERT model based on the example mentioned in issue #2092.

## Before submitting
- [x] Did you make sure to update the documentation with your changes?
- [ ]  This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you write any new necessary tests?

## Who can review?
@IlyasMoutawwakil

### CODE REVIEW

File: optimum/exporters/tasks.py [hunk 1]
```diff
@@ -431,6 +431,15 @@ class TasksManager:
             onnx="BertOnnxConfig",
             tflite="BertTFLiteConfig",
         ),
+        "rembert": supported_tasks_mapping(
+            "text-generation",
```
Comment (IlyasMoutawwakil) [inline optimum/exporters/tasks.py:435]:
```suggestion
            # the logic for text-generation is not supported for BERT
            # "text-generation",
```

File: optimum/exporters/tasks.py [hunk 2]
```diff
@@ -431,6 +431,15 @@ class TasksManager:
...
+            "fill-mask",
```
Comment (IlyasMoutawwakil) [inline optimum/exporters/tasks.py:436]:
```suggestion
            "fill-mask",
            "feature-extraction",
```

File: tests/onnxruntime/test_modeling.py [hunk 3]
```diff
@@ -2326,6 +2331,7 @@ class ORTModelForCausalLMIntegrationTest(ORTModelTestMixin):
         "llama",
         "mistral",
         "opt",
+        "rembert",
```
Comment (IlyasMoutawwakil) [inline tests/onnxruntime/test_modeling.py:2334]:
```suggestion
```

**Review by echarlaix — APPROVED** [review]
LGTM, thanks for your contribution @mlynatom

**Pull request merged**